tests/PresenterTest.php

```diff
@@ -497,6 +514,40 @@ public function a_presenter_can_specify_attributes_to_show_in_json_or_array_outp
 
         $this->assertEquals($desired, (string) $presentedModel);
     }
+
+    /** @test */
+    public function it_can_be_array_accessed()
+    {
+        $sampleModel = TestModel::create([
+            'first_name' => 'David',
+            'last_name' => 'Hemphill',
+            'created_at' => '2015-10-14 12:00:00',
+            'updated_at' => '2015-10-14 12:00:00',
+        ]);
+
+        $presenter = $sampleModel->present(SamplePresenter::class);
+
+        $this->assertEquals('David', $presenter['first_name']);
+
+        $this->assertEquals('David Lee Hemphill', $presenter['full_name']);
+    }
+
+    /** @test */
+    public function it_cannot_be_written_to_via_array_access()
+    {
+        $sampleModel = TestModel::create([
+            'first_name' => 'David',
+            'last_name' => 'Hemphill',
+            'created_at' => '2015-10-14 12:00:00',
+            'updated_at' => '2015-10-14 12:00:00',
+        ]);
+
+        $presenter = $sampleModel->present(SamplePresenter::class);
+
+        $this->setExpectedException(\BadMethodCallException::class);
```

Comment:
`setExpectedException` was deprecated in PHPUnit 6. Think you could migrate this to the annotation? Also, if you'd import `BadMethodCallException` that'd be rad.